task_DEV_4/task_DEV_4/Rectangle.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace task_DEV_4
+{
+    class Rectangle
+    {
+
+        public int height;
+        public int width;
+
+        public Rectangle()
+        {
+
+        }
+
+        public Rectangle(int height, int width)
+        {
+            this.height = height;
+            this.width = width;
+        }
+
+        public void SettingRectangleValues(int height, int width)
+        {
+            this.height = height;
+            this.width = width;
+        }
+
+        public int CalculatingThePerimeterOfARectangle()
```

Comment:
The first word in the name of the method should be verb.
Usually, articles are not written in the name of methods.
Better to name `CalculatePerimeter`. It is clear that perimeter of rectangle because this method is inside of class Rectangle.

Reply:
done